### PR TITLE
[Localization] Missing translations in battler-tags of most languages

### PR DESCRIPTION
**Pull request by Adri1**
## What are the changes the user will see?
Have these translated

## Why am I making these changes?
To have it translated

## What are the changes from a developer perspective?
None

## Checklist
- [X] **I'm using `beta` as my base branch**
- [X] There is no overlap with another PR?
- [X] The PR is self-contained and cannot be split into smaller PRs?
- [X] Have I provided a clear explanation of the changes?

### CODE REVIEW

File: src/locales/pt_BR/battler-tags.json [hunk 1]
```diff
@@ -73,5 +73,9 @@
   "tarShotOnAdd": "{{pokemonNameWithAffix}} tornou-se mais fraco ao fogo!",
   "substituteOnAdd": "{{pokemonNameWithAffix}} colocou um substituto!",
   "substituteOnHit": "O substituto tomou o dano pelo {{pokemonNameWithAffix}}!",
-  "substituteOnRemove": "O substituto de {{pokemonNameWithAffix}} desbotou!"
+  "substituteOnRemove": "O substituto de {{pokemonNameWithAffix}} desbotou!",
+  "tormentOnAdd": "{{pokemonNameWithAffix}} was subjected to torment!",
+  "tauntOnAdd": "{{pokemonNameWithAffix}} fell for the taunt!",
+  "imprisonOnAdd": "{{pokemonNameWithAffix}} sealed the opponents move(s)!",
+  "autotomizeOnAdd": "{{pokemonNameWithAffix}} became nimble!"
```
Comment (josericardo-fo) [inline src/locales/pt_BR/battler-tags.json:80]:
```suggestion
  "tormentOnAdd": "{{pokemonNameWithAffix}} foi submetido a um tormento!",
  "tauntOnAdd": "{{pokemonNameWithAffix}} caiu na provocação!",
  "imprisonOnAdd": "{{pokemonNameWithAffix}} selou o(s) movimento(s) do oponente!",
  "autotomizeOnAdd": "{{pokemonNameWithAffix}} tornou-se ágil!"
```

File: src/locales/ja/battler-tags.json [hunk 2]
```diff
@@ -71,5 +71,9 @@
   "disabledOnAdd": "{{pokemonNameWithAffix}}の\n{{moveName}}\nを　封じこめた！",
   "disabledLapse": "{{pokemonNameWithAffix}}の\nかなしばりが　解けた！",
   "tarShotOnAdd": "{{pokemonNameWithAffix}}は　ほのおに　弱くなった！",
-  "shedTailOnAdd": "{{pokemonNameWithAffix}}は\nしっぽを　切って　みがわりにした！"
+  "shedTailOnAdd": "{{pokemonNameWithAffix}}は\nしっぽを　切って　みがわりにした！",
+  "tormentOnAdd": "{{pokemonNameWithAffix}} was subjected to torment!",
+  "tauntOnAdd": "{{pokemonNameWithAffix}} fell for the taunt!",
+  "imprisonOnAdd": "{{pokemonNameWithAffix}} sealed the opponents move(s)!",
+  "autotomizeOnAdd": "{{pokemonNameWithAffix}} became nimble!"
```
Comment (Chapybara-jp) [inline src/locales/ja/battler-tags.json:78]:
```suggestion
  "tormentOnAdd": "{{pokemonNameWithAffix}} は　いちゃもんを　つけられた！",
  "tauntOnAdd": "{{pokemonNameWithAffix}} は　挑発に　乗ってしまった！",
  "imprisonOnAdd": "{{pokemonNameWithAffix}} は　相手の技を　封印した！",
  "autotomizeOnAdd": "{{pokemonNameWithAffix}} は　身軽になった！"
```

File: src/locales/ko/battler-tags.json [hunk 3]
```diff
@@ -74,5 +74,9 @@
   "shedTailOnAdd": "{{pokemonNameWithAffix}}[[는]] 꼬리를 잘라 대타로 삼았다!",
   "substituteOnAdd": "{{pokemonNameWithAffix}}의\n대타가 나타났다!",
   "substituteOnHit": "{{pokemonNameWithAffix}}[[를]] 대신하여\n대타가 공격을 받았다!",
-  "substituteOnRemove": "{{pokemonNameWithAffix}}의\n대타는 사라져 버렸다..."
+  "substituteOnRemove": "{{pokemonNameWithAffix}}의\n대타는 사라져 버렸다...",
+  "tormentOnAdd": "{{pokemonNameWithAffix}} was subjected to torment!",
+  "tauntOnAdd": "{{pokemonNameWithAffix}} fell for the taunt!",
+  "imprisonOnAdd": "{{pokemonNameWithAffix}} sealed the opponents move(s)!",
+  "autotomizeOnAdd": "{{pokemonNameWithAffix}} became nimble!"
```
Comment (EnochG1) [inline src/locales/ko/battler-tags.json:81]:
```suggestion
  "tormentOnAdd": "{{pokemonNameWithAffix}}[[는]]\n트집을 잡혔다!",
  "tauntOnAdd": "{{pokemonNameWithAffix}}[[는]]\n도발에 넘어가 버렸다!",
  "imprisonOnAdd": "{{pokemonNameWithAffix}}[[는]]\n상대의 기술을 봉인했다!",
  "autotomizeOnAdd": "{{pokemonNameWithAffix}}의\n몸이 가벼워졌다!"
```

**Comment by frutescens** [comment]
Unique Move-Disabling Messages:
Torment

image
Taunt

image
Heal Block

image
Imprison

image

Some Notes...
- Smogon/Mainline Games have the taunt / torment activation in lowercase and move references tend to be capitalized so I wasn't sure if I should derive the name from the move itself. Let me know if I should change anything.

File: src/locales/zh_CN/battler-tags.json [hunk 4]
```diff
@@ -74,5 +74,9 @@
   "shedTailOnAdd": "{{pokemonNameWithAffix}}\n断掉尾巴并将其作为替身了！",
   "substituteOnAdd": "{{pokemonNameWithAffix}}的\n替身出现了！",
   "substituteOnHit": "替身代替{{pokemonNameWithAffix}}\n承受了攻击！",
-  "substituteOnRemove": "{{pokemonNameWithAffix}}的\n替身消失了……"
+  "substituteOnRemove": "{{pokemonNameWithAffix}}的\n替身消失了……",
+  "tormentOnAdd": "{{pokemonNameWithAffix}} was subjected to torment!",
+  "tauntOnAdd": "{{pokemonNameWithAffix}} fell for the taunt!",
+  "imprisonOnAdd": "{{pokemonNameWithAffix}} sealed the opponents move(s)!",
+  "autotomizeOnAdd": "{{pokemonNameWithAffix}} became nimble!"
```
Comment (40chyan) [inline src/locales/zh_CN/battler-tags.json:81]:
```suggestion
"tormentOnAdd": "{{pokemonNameWithAffix}}遭到了无理取闹！",
"tauntOnAdd": "{{pokemonNameWithAffix}}中了挑衅！",
"imprisonOnAdd": "{{pokemonNameWithAffix}}封印了对手的招式！",
"autotomizeOnAdd": "{{pokemonNameWithAffix}}变得身轻如燕了！"
```

**Review by CodeTappert — DISMISSED** [review]

**Review by Chapybara-jp — APPROVED** [review]

**Pull request closed without merging**